src/Console/Command/FixCommand.php

```diff
@@ -318,10 +315,24 @@ protected function execute(InputInterface $input, OutputInterface $output)
             ->resolve()
         ;
 
+        $reporter = ReporterFactory::create()
+            ->registerBuiltInReporters()
+            ->getReporter($resolver->getFormat())
+        ;
+
+        $stdErr = $output instanceof ConsoleOutputInterface
+            ? $output->getErrorOutput()
+            : ('txt' === $reporter->getFormat() ? $output : null)
```

Comment:
I find nested ternary statements nice to read as long as they are properly used.